### PR TITLE
chore: upgrade nuxt & fix some warnings

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Context

- [x] upgrade nuxt to the latest version
- [x] I noticed that after upgrading to the latest version of nuxt, there are some warnings on console related svg-loader. then, I moved some of svgs to fix the warnings
- [x] remove sentry related packages at the moment
- [x] upgrade i18n package to the stable version
- [x] fix console warnings related to using $fetch instead. and some other warnings

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

### CODE REVIEW

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 9.92 USD/DOT ~ 5.04 $DOT
🧗 [1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p](https://polkadot.subscan.io/account/1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)
🔗 [0xa9fa28619b3b77b94039b630cfab802502857238fa4628285f6425825c6567ff](https://polkadot.subscan.io/extrinsic/0xa9fa28619b3b77b94039b630cfab802502857238fa4628285f6425825c6567ff)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
when it's everything ok, @prury can merge

**Comment by yangwao** [comment]
oh I see v3.11, sweet! 
- https://twitter.com/nuxt_js/status/1769289366378008696

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
@preschian i'm still getting the locale error when building: 

image

the other warns/errors are not showing anymore, nice!

**Comment by preschian** [comment]
> i'm still getting the locale error when building:

yes, for the rest of the warnings, I still don't know yet how to fix it. there are also still leftover browser console warnings such as https://github.com/kodadot/nft-gallery/issues/9662

**Comment by prury** [comment]
> > i'm still getting the locale error when building:
> 
> Yes, I still don't know how to fix the rest of the warnings. There are also still leftover browser console warnings, such as https://github.com/kodadot/nft-gallery/issues/9662
> 
> I changed the PR title to avoid confusion

Thank you!

File: components/landing/topCollections/utils/useTopCollections.ts
Comment (prury) [inline components/landing/topCollections/utils/useTopCollections.ts:1]: hitting back when accessing item from top collections won't load their cards anymore: 
https://github.com/kodadot/nft-gallery/assets/36627808/1bbfca91-8468-4e65-925d-d12f8f17334f

Comment (preschian) [reply components/landing/topCollections/utils/useTopCollections.ts:1]: updated

**Comment by prury** [comment]
thank you for the test! mergin!

**Review by prury — APPROVED** [review]

**Pull request merged**